e2e-tests/pages/insurance/your-buyer/index.js

```diff
@@ -1,3 +1,4 @@
 import companyOrOrganisationPage from './companyOrOrganisation';
+import buyerFinancialInformationPage from './buyerFinancialAccounts';
 
-export { companyOrOrganisationPage };
+export { companyOrOrganisationPage, buyerFinancialInformationPage };
```

Comment:
`buyerFinancialInformation` vs `buyerFinancialAccounts` ?

Reply:
the page/url is buyer-financial-information but the field is buyerFinancialAccounts.  Should maybe keep it as buyerFinancialInformation?